pkg/verifier/aws/entry_point.go

```diff
@@ -177,7 +181,7 @@ func (a *AwsVerifier) ValidateEgress(vei verifier.ValidateEgressInput) *output.O
 		userDataVariables["DELAY"] = "60"
 	}
 
-	unencodedUserData, err := vei.Probe.GetExpandedUserData(userDataVariables)
+	unencodedUserData, err := vei.Probe.GetExpandedUserData(userDataVariables, userDataTemplate)
```

Comment:
So this has us providing the same value of userDataTemplate to every probe, yeah? My first thought is that different probes need to use different user-data, but then I saw that your LegacyProbe just ignores the userDataTemplate param. So it works, but it's not immediately clear to somebody scanning through the code why this would work. IOW, this needs comments at the very least, but let's have a larger discussion about this first